modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -45,7 +45,7 @@ has initialized.
 [source,terminal]
 ----
 $ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-instance-group --instance-group-zone=${ZONE_0}
-$ gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
-$ gsutil rb gs://${INFRA_ID}-bootstrap-ignition
+$ gsutil rm gs://${INFRA_ID}-bootstrap-ig/bootstrap.ign
+$ gsutil rb gs://${INFRA_ID}-bootstrap-ig
```

Comment:
The bucket name is `${INFRA_ID}-bootstrap-ignition`, so the 2 lines should remain unchanged. Thanks!

Reply:
Reverted

Reply:
Please keep the previous line 48 and line 49, thanks! 

```
$ gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
$ gsutil rb gs://${INFRA_ID}-bootstrap-ignition

Reply:
Reverted